packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -1,31 +1,115 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
+import Button from 'terra-button';
+import IconClose from 'terra-icon/lib/icon/IconClose';
+import NavItems from './components/nav-items/NavItems';
+import NavLogo from './components/nav-logo/NavLogo';
+import NavProfile from './components/nav-profile/NavProfile';
+import QuickLink from './components/quick-links/QuickLink';
+import QuickLinks from './components/quick-links/QuickLinks';
 import styles from './Nav.scss';
 
 const cx = classNames.bind(styles);
 
+const navItemShape = {
+  /**
+   * The path the nav item should lead to.
+   */
+  uri: PropTypes.string,
```

Comment:
i agree, i think this and text, since we are not allowing icons, should be required. I will add this to navitems and quicklink